linux_os/guide/system/permissions/files/no_files_unowned_by_user/oval/shared.xml

```diff
@@ -17,14 +17,12 @@
   </unix:file_state>
 
   <local_variable id="file_permissions_unowned_userid_list" comment="List of valid user ids" datatype="int" version="1">
-    <object_component item_field="subexpression" object_ref="file_permissions_unowned_userid_list_object" />
+    <object_component item_field="user_id" object_ref="file_permissions_unowned_userid_list_object" />
   </local_variable>
 
-  <ind:textfilecontent54_object id="file_permissions_unowned_userid_list_object" version="1">
-    <ind:filepath>/etc/passwd</ind:filepath>
-    <ind:pattern operation="pattern match">^[^:]+:[^:]+:([\d]+):[\d]+:[^:]*:[^:]+:[^:]*$</ind:pattern>
-    <ind:instance operation="greater than or equal" datatype="int">1</ind:instance>
-  </ind:textfilecontent54_object>
+  <unix:password_object id="file_permissions_unowned_userid_list_object" version="1">
```

Comment:
``password_object`` appears to only accept a single user name, not a list. Will the rule iterate through all elements of ``file_permissions_unowned_userid_list_object ``?

Ref https://oval.mitre.org/language/version5.10/ovaldefinition/documentation/unix-definitions-schema.html
``
A password object consists of a single username entity that identifies the user(s) whose password is to be evaluated.
``

Reply:
> `password_object` appears to only accept a single user name, not a list. Will the rule iterate through all elements of `file_permissions_unowned_userid_list_object `?

Each user id is stored in a local_variable just like it is with textfilecontent54_object.

> Ref https://oval.mitre.org/language/version5.10/ovaldefinition/documentation/unix-definitions-schema.html
> `A password object consists of a single username entity that identifies the user(s) whose password is to be evaluated.`

The language is confusing, but it means that there is no other element other than "username" for the "password_object".

You can run the following before and after the change to double check me:
```PYTHONPATH=`./.pyenv.sh` utils/testoval.py linux_os/guide/system/permissions/files/no_files_unowned_by_user/oval/shared.xml```
I noticed no difference in the results.

Reply:
Thanks!